VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -1,12 +1,10 @@
-<android.support.v4.widget.DrawerLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
since we aren't using the `app` namespace now, so, autoformat seems to have moved it to the same line. Looks fine currently I guess.